src/main/java/ogc/rs/apiserver/router/routerbuilders/StacRouterBuilder.java

```diff
@@ -97,13 +97,27 @@ void addImplSpecificRoutes() {
         .handler(apiServerVerticle::buildResponse)
         .failureHandler(failureHandler);
 
-      routerBuilder
-              .operation(STAC_ITEM_BY_ID_API)
-              .handler(stacItemByIdAuthZHandler)
-              .handler(apiServerVerticle::getStacItemById)
-              .handler(apiServerVerticle::putCommonResponseHeaders)
-              .handler(apiServerVerticle::buildResponse)
-              .failureHandler(failureHandler);
+    routerBuilder
+        .operation(STAC_ITEM_BY_ID_API)
+        .handler(stacItemByIdAuthZHandler)
+        .handler(apiServerVerticle::getStacItemById)
+        .handler(apiServerVerticle::putCommonResponseHeaders)
+        .handler(apiServerVerticle::buildResponse)
+        .failureHandler(failureHandler);
+
+    routerBuilder
+        .operation(STAC_ITEMS_POST_API)
```

Comment:
Add authZ handlers for both APIs - I think you can use the same thing the STAC Collection transaction APIs use.